### DISCUSSION TITLE
KEDA scale up/down step

### DISCUSSION THREAD

**Discussion by patrykgiertych** [kedacore/keda#6027] [Q&A / Need Help]
Hi, I'm looking for a way to setup by how many pods the ScaledObject will scale up/down, it seems to be always scaling by 2 at a time, I would like to set it to scale one by one.

**Comment by JorTurFer** [comment]
Hello
You can, just configure the HPA advanced section 😄 
https://keda.sh/docs/2.15/reference/scaledobject-spec/#horizontalpodautoscalerconfig
